### PR TITLE
Bump bundler from 2.2.27 to 2.2.28

### PR DESCRIPTION
**Pull request by dev-yakuza**
- https://github.com/rubygems/rubygems/releases/tag/bundler-v2.2.28
- https://github.com/rubygems/rubygems/blob/bundler-v2.2.28/bundler/CHANGELOG.md
- https://my.diffend.io/gems/bundler/2.2.27/2.2.28

### CODE REVIEW

File: Gemfile [hunk 1]
```diff
@@ -1,6 +1,6 @@
 source 'https://rubygems.org'
 
 gem 'aufgaben'
-gem 'bundler', '2.2.26'
+gem 'bundler', '2.2.28'
```
Comment (ybiquitous) [inline Gemfile:4]: The current version is `2.2.27`, isn't it?

https://github.com/sider/devon_rex/blob/ef075671e6a1d66a21196c14304cc3cadddfa404/Gemfile#L4

Comment (dev-yakuza) [reply Gemfile:4]: I just created the branch from old...now I try to rebase...

Comment (suetsuguk) [reply Gemfile:4]: CHANGELOG and Gemfile seem to be old files. Sync with the master. Thank you.

File: CHANGELOG.md [hunk 2]
```diff
@@ -6,8 +6,9 @@ All notable changes to this project will be documented in this file.
 
 [Full diff](https://github.com/sider/devon_rex/compare/2.45.10...HEAD)
 
-- Bump bundler from 2.2.26 to 2.2.27 [#653](https://github.com/sider/devon_rex/pull/653)
+- Bump bundler from 2.2.26 to 2.2.28 [#653](https://github.com/sider/devon_rex/pull/653) [#664](https://github.com/sider/devon_rex/pull/664)
 - Bump debian from buster-20210816 to buster-20210902 [#643](https://github.com/sider/devon_rex/pull/643)
+- Bump dotnet/sdk from 3.1.412-buster to 3.1.413-buster [#659](https://github.com/sider/devon_rex/pull/659)
```
Comment (dev-yakuza) [inline CHANGELOG.md:11]: Something is wrong...I just close this PR and re-create...

**Comment by dev-yakuza** [comment]
I just re-create a PR about `bundler`.

- https://github.com/sider/devon_rex/pull/665

**Pull request closed without merging**